Makefile

```diff
@@ -39,9 +39,9 @@ minikube-dashboard:
 	minikube --profile minikube-wbaas dashboard
 
 .PHONY: minikube-tunnel
-minikube-tunnel: # @HELP Open a tunnel to the local cluster and expose it on an IP on the host system
+minikube-tunnel: # @HELP Open a tunnel to the local ingress via kubectl and expose it at port 80 on the host system
```

Comment:
maybe it would make sense to introduce a new target to be called like `make minikube-port-forward` or something?